### DISCUSSION TITLE
Large Files

### DISCUSSION THREAD

**Discussion by nthoman** [pnp/syntex-samples#3] [General]
I have a 2000 page pdf file that I want to extract the last few values from this document but I can't because the training tool wont let me jump to the end of the file to mark my label. 

PS... I know... its a "Report" that I want to get rid of but it's for audit purposes.

**Comment by jameseccles** [comment]
2000 pages would be outside of our current file size limits (64k characters for office files, 20 pages for OCR content). https://docs.microsoft.com/en-gb/microsoft-365/contentunderstanding/document-understanding-overview#file-limitations

Would breaking out a subset of pages from the PDF be an acceptable workaround?

**Reply by nthoman** [reply]
Maybe.... I need to get back to the business and actually find the purpose of this report. I doubt anything is actually looking through a 2000 page report on the regular.
